nixos/modules/services/audio/alsa.nix

```diff
@@ -83,7 +83,18 @@ in
 
   ###### implementation
 
-  config = mkIf config.sound.enable {
+  config = mkMerge [{
+
+    warnings = optional (options.sound.enable.highestPrio > 1000) ''
+      You don't have `sound.enable` explicitly enabled of disabled. It was
+      enabled by default before a43e33d0e48b2284ac3a2222d7f1965cef66f5e2 and
+      became disabled by default after e349ccc77febd45abbd14be14f7de123ec4a4da2.
```

Comment:
There is no point in including commit hashes in warnings.

Reply:
I'm open to better suggestions. I don't like being vague like "some commit before 18.03" (which was another option I could come up with) when I can just give the hash (I like doing `git show` on random hashes I see mentioned in random PRs and comments), but I won't argue against writing whatever else you want if that gets the message across.